psydac/core/bsplines.py

```diff
@@ -58,11 +60,14 @@ def find_span( knots, degree, x ):
         Knot span index.
 
     """
+
+
```

Comment:
Useless empty lines change. Please remove

Reply:
Fixed in https://github.com/pyccel/psydac/commit/c9eac22256b58bb6b2138192ad3cc5766ae2d143